### PR TITLE
dev-python/pyswisseph: version bump to 2.05.01

### PR DESCRIPTION
**Pull request by lluixhi**
This version hasn't been uploaded to pypi, but the author's most recent code upload was in May of last year. It appears to be compatible with python 3.6

### CODE REVIEW

**Comment by mgorny** [comment]
Please try contacting upstream about this. I wouldn't feel secure assuming version update implies a release, especially that upstream did not create any tag. If they do not confirm this release (and I suppose they would do the release tarball as well in that case), please mark this as snapshot, i.e. appending the *_preDATE* to version.

**Comment by mgorny** [comment]
Ping. Any news?

**Comment by lluixhi** [comment]
No response from the maintainer.
I'll append the date.

File: dev-python/pyswisseph/pyswisseph-2.05.01_pre20160529.ebuild [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	SRC_URI="https://github.com/astrorigin/${PN}/archive/${COMMIT_ID}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/${PN}-${COMMIT_ID}"
+fi
+
+LICENSE="GPL-2+"
+SLOT="0"
+
+DEPEND=""
+RDEPEND="${DEPEND}"
```
Comment (mgorny) [inline dev-python/pyswisseph/pyswisseph-2.05.01_pre20160529.ebuild]: Hmm, I see that it has an optional dep on libswe (+ pkg-config); but I think we don't have that lib, so it's ok for now.

**Pull request closed without merging**